apps/api/src/app/environments-v1/usecases/construct-framework-workflow/construct-framework-workflow.usecase.ts

```diff
@@ -139,6 +144,10 @@ export class ConstructFrameworkWorkflow {
       },
       {
         payloadSchema: PERMISSIVE_EMPTY_SCHEMA,
+        name: dbWorkflow.name,
+        description: dbWorkflow.description,
+        tags: dbWorkflow.tags,
+        severity: dbWorkflow.severity,
```

Comment:
pass workflow options, which are used by the framework to build the discovered workflow